java/arcs/core/util/platform/native/PlatformDuration.kt

```diff
@@ -37,5 +38,9 @@ class PlatformDuration {
         @Suppress("UNUSED_PARAMETER")
         fun ofDays(days: Long): PlatformDuration =
             TODO("Add support for ArcsDuration in Kotlin Native")
+
+        @Suppress("UNUSED_PARAMETER")
+        fun ofHours(hours: Long): PlatformDuration =
+            TODO("Add support for ArcsDuration in Kotlin Native")
```

Comment:
Would be great to have a tracking bug for these, you can include it in the TODO

Reply:
Filed https://b.corp.google.com/issues/169213588 to track adding support for BigInt, ArcsInstant and ArcsDuration to js+native.

Hopefully they're only needed temporarily as Kotlin comes along.